app/src/colorpalettewidget.cpp

```diff
@@ -143,6 +151,45 @@ void ColorPaletteWidget::removeItem()
     clickRemoveColorButton();
 }
 
+void ColorPaletteWidget::replaceLineColorSingle()
+{
+    mReplaceLineColorAll = false;
+    replaceLineColor();
+}
+
+void ColorPaletteWidget::replaceLineColor()
+{
+    if (mMultipleSelected) { return; }
+
+    const ColourRef colourRef = editor()->object()->getColour(ui->colorListWidget->currentRow());
+    QRgb newlinecolor = colourRef.colour.rgb();
+
+    LayerBitmap *layerbitmap = static_cast<LayerBitmap*>(editor()->layers()->currentLayer());
+
+    if (!mReplaceLineColorAll)
+    {
+        if (layerbitmap->keyExists(editor()->currentFrame()))
+        {
+            layerbitmap->getBitmapImageAtFrame(editor()->currentFrame())->fillNonAlphaPixels(newlinecolor);
+            editor()->updateFrame(editor()->currentFrame());
+        }
+    }
+    else
+    {
+        int org = editor()->currentFrame();
```

Comment:
The convention here is usually to call it currentFrame

Reply:
I'll change org to currentFrame.